packages/@sanity/form-builder/src/inputs/Array/ItemValue.js

```diff
@@ -136,13 +137,10 @@ export default class RenderItemValue extends React.Component<Props> {
 
     const memberType = this.getMemberType() || {}
 
-    // Reset level if a full screen modal
-    const level = options.editModal === 'fullscreen' ? 0 : 2
```

Comment:
Have we removed support for any of these options now? If so, we need to update the docs.

Reply:
Not yet. But we set the level directly on the component

Reply:
Ah, cause we're just replacing the default dialog now. Got it, thanks 👍